pkg/controller/utils/utils.go

```diff
@@ -19,10 +19,21 @@ const (
 	Finalizer = "finalizer.aws.managed.openshift.io"
 	WaitTime  = 25
 
-	// EnvDevMode is the name of the env var we set to run locally and to skip
-	// initialization procedures that will error out and exit the operator.
-	// ex: `FORCE_DEV_MODE=local operatorsdk up local`
-	EnvDevMode = "FORCE_DEV_MODE"
+	// envDevMode is the name of the env var we set to indicate we're running in a development
+	// environment vs. production. Set it to one of the DevMode* consts defined below.
+	// For example:
+	// * Running locally: `FORCE_DEV_MODE=local operator-sdk up local`
+	// * Running in a cluster:
+	//   . Edit deploy/operator.yaml. Under .spec.template.spec.env, add:
+	//         - name: FORCE_DEV_MODE
+	//           value: cluster
+	//   . `oc apply` all the YAML files in deploy/, including the updated operator.yaml.
+	envDevMode = "FORCE_DEV_MODE"
+
+	// MetricsPort is the port at which we'll serve metrics
+	MetricsPort               = "8080"
+	// MetricsPath is the URI path at which we'll serve metrics
+	MetricsPath               = "/metrics"
```

Comment:
Moved these in from main.go mainly to make them easier to refer to from the comments below.